src/test/java/com/github/onsdigital/dpjavadurationparse/ParseDurationTest.java

```diff
@@ -0,0 +1,115 @@
+package com.github.onsdigital.dpjavadurationparse;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
```

Comment:
This import statement is not used (InvocationTargetException) in the class so should be removed